sample.env

```diff
@@ -23,6 +23,8 @@ AZURE_FORM_RECOGNIZER_KEY=
 #  Open Telemetry Config (used within the simulator)
 OTEL_SERVICE_NAME=aoai-api-simulator-local-dev
 OTEL_METRIC_EXPORT_INTERVAL=10000
+OTEL_EXPORTER_OTLP_ENDPOINT=http://host.docker.internal:4317
```

Comment:
If this is set and otel-lgtm hasn't been started then there are a bunch of errors that are emitted:

```
WARNING:opentelemetry.exporter.otlp.proto.grpc.exporter:Transient error StatusCode.DEADLINE_EXCEEDED encountered while exporting logs to host.docker.internal:4317, retrying in 1s.
```

I think it's good to include here as an example but I'd prefer that we comment it out.

Reply:
Also, is there any reason that this is set to `host.docker.internal` rather than just `localhost`?